sphinx/ext/autosummary/generate.py

```diff
@@ -197,6 +197,21 @@ def get_members(obj: Any, types: Set[str], include_public: List[str] = [],
             get_members(obj, {'class'}, imported=imported_members)
         ns['exceptions'], ns['all_exceptions'] = \
             get_members(obj, {'exception'}, imported=imported_members)
+
+        # Find module attributes with docstrings
+        attrs, public = [], []
+        from sphinx.pycode import ModuleAnalyzer, PycodeError
```

Comment:
Could you move this line to the top of this module, please? Basically we go importing there if no reason.